TopMovies/API/MovieTarget.swift

```diff
@@ -8,6 +8,7 @@ import ReSwift
 import Moya
 
 class MovieTarget: StoreSubscriber {
+  @Inject private var mainStore: MainStore
```

Comment:
Why do you have it here?

Reply:
> Why do you have it here?

I need to receive a configured API Key from the store, that's why i have it in the Target

Reply:
Ah, got it.